index.js

```diff
@@ -6,11 +6,15 @@ var YAML = require('yamljs');
 /**
  *
  * @param {number|string} performanceYear - four digit year
- * @return {Array<Benchmark>} - benchmarks data for given year
+ * @return {Array<Benchmark>|undefined} - benchmarks data for given year
  */
 exports.getBenchmarksData = function(performanceYear) {
-  return JSON.parse(
-    fs.readFileSync(path.join(__dirname, 'benchmarks', performanceYear + '.json')));
+  try {
+    return JSON.parse(
+      fs.readFileSync(path.join(__dirname, 'benchmarks', performanceYear + '.json')));
+  } catch (error) {
+    return;
```

Comment:
If this is an unrecoverable error, as a library, we should throw a documented error and let the application handle the failure.

Tangentially, is this the first PR that introduces a dependency on `fs`? Do we need to make that clear somewhere? If there's value in a serverless use of this library, maybe we should consider a build step to bake benchmarks into the exported file.